pom.xml

```diff
@@ -2405,6 +2423,7 @@
         <hudi.flink.module>hudi-flink1.14.x</hudi.flink.module>
         <flink.bundle.version>1.14</flink.bundle.version>
         <orc.flink.version>1.5.6</orc.flink.version>
+        <flink.avro.version>1.10.0</flink.avro.version>
```

Comment:
Can we use 1.11.1 here, there will be a NoSuchMethodError with 1.10.0
```
java.lang.NoSuchMethodError: org.apache.hudi.org.apache.avro.specific.SpecificRecordBuilderBase.<init>(Lorg/apache/hudi/org/apache/avro/Schema;Lorg/apache/hudi/org/apache/avro/specific/SpecificData;)V
```

Reply:
you can fire a PR to fix it.